GitUI/RepoObjectsTree/RepoObjectsTree.Designer.cs

```diff
@@ -0,0 +1,580 @@
+			this.branchSearchPanel.Name = "branchSearchPanel";
+			this.branchSearchPanel.RowStyles.Add(new System.Windows.Forms.RowStyle());
+			this.branchSearchPanel.Size = new System.Drawing.Size(294, 32);
+			this.branchSearchPanel.TabIndex = 4;
+			// 
+			// lblSearchBranch
+			// 
+			this.lblSearchBranch.Anchor = System.Windows.Forms.AnchorStyles.Left;
+			this.lblSearchBranch.AutoSize = true;
+			this.lblSearchBranch.Location = new System.Drawing.Point(3, 9);
+			this.lblSearchBranch.Name = "lblSearchBranch";
+			this.lblSearchBranch.Size = new System.Drawing.Size(44, 13);
+			this.lblSearchBranch.TabIndex = 0;
+			this.lblSearchBranch.Text = "Search:";
+			// 
+			// btnSearch
+			// 
+			this.btnSearch.Anchor = System.Windows.Forms.AnchorStyles.Left;
+			this.btnSearch.AutoSize = true;
+			this.btnSearch.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
+			this.btnSearch.Image = global::GitUI.Properties.Resources.IconFind;
+			this.btnSearch.Location = new System.Drawing.Point(233, 3);
+			this.btnSearch.Name = "btnSearch";
+			this.btnSearch.Padding = new System.Windows.Forms.Padding(2);
+			this.btnSearch.Size = new System.Drawing.Size(26, 26);
+			this.btnSearch.TabIndex = 2;
+			this.btnSearch.UseVisualStyleBackColor = true;
+			this.btnSearch.Click += new System.EventHandler(this.OnBtnSearchClicked);
+			// 
+			// btnSettings
+			// 
+			this.btnSettings.Anchor = System.Windows.Forms.AnchorStyles.Left;
+			this.btnSettings.AutoSize = true;
+			this.btnSettings.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
+			this.btnSettings.ContextMenuStrip = this.menuSettings;
+			this.btnSettings.Image = global::GitUI.Properties.Resources.Settings;
+			this.btnSettings.Location = new System.Drawing.Point(265, 3);
+			this.btnSettings.Name = "btnSettings";
+			this.btnSettings.Padding = new System.Windows.Forms.Padding(2);
+			this.btnSettings.Size = new System.Drawing.Size(26, 26);
+			this.btnSettings.TabIndex = 3;
+			this.btnSettings.UseVisualStyleBackColor = true;
+			this.btnSettings.Click += new System.EventHandler(this.OnBtnSettingsClicked);
+			// 
+			// menuSettings
+			// 
+			this.menuSettings.Items.AddRange(new System.Windows.Forms.ToolStripItem[] {
+            this.showTagsToolStripMenuItem});
+			this.menuSettings.Name = "menuSettings";
+            this.menuSettings.Size = new System.Drawing.Size(137, 26);
+			// 
+			// showTagsToolStripMenuItem
+			// 
+			this.showTagsToolStripMenuItem.CheckOnClick = true;
+			this.showTagsToolStripMenuItem.Name = "showTagsToolStripMenuItem";
+            this.showTagsToolStripMenuItem.Size = new System.Drawing.Size(136, 22);
+			this.showTagsToolStripMenuItem.Text = "Show &tags";
+			this.showTagsToolStripMenuItem.Click += new System.EventHandler(this.ShowTagsToolStripMenuItem_Click);
+			// 
+			// RepoObjectsTree
+			// 
+			this.Controls.Add(this.repoTreePanel);
+			this.MinimumSize = new System.Drawing.Size(300, 0);
```

Comment:
Can we change the minimum width to 100?

Reply:
what is the reason to change it? It doesn't restrict the panel to have a minimum width of 300, you can drag the splitter as you wish and even hide it completely using the toggle.
The 100 doesn't work well with the search text box (the search button overlaps the box), seems 190 is the minimum.

Reply:
When the minimum width is set 300 and you try to make the left panel shorter then the buttons are hidden:
![gitext](https://user-images.githubusercontent.com/1003909/36946651-21b80296-1fc0-11e8-9be3-661ff07b2941.png)